src/pages/Build/BuildPage.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-	<www-page>
+	<www-page
+		:header-theme="greenHeader"
```

Comment:
The entire "Build" section can move to a the new default light header. I think the only outlying page is the one that might have a users api credentials but that's ok for now.

Reply:
We can also just let it be and wait until the full header refresh. Don't let one block the merge.